### DISCUSSION TITLE
How can figure out whether a detail gird on a master form has any item or not?

### DISCUSSION THREAD

**Discussion by amirkian** [serenity-is/Serenity#5493] [General]
whenever a grid form is empty in a master form ,I want to write something in console.

**Comment by reach2rv** [comment]
in master dialog validate entity
validateEntity(row: XYZMasterRow, id: number) {
            if (row.ItemDetails.length == 0) {
                Q.alert("No Items added, At least add one entry. Can not Save! ")
                return false;
            }
}

**Reply by amirkian** [reply]
thanks; where I should insert it and what do you mean by XYZMasterRow ?

**Reply by reach2rv** [reply]
In master dialog
